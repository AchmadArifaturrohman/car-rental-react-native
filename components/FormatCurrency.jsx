import React from "react";

export default function FormatCurrency({ amount }) {
  const formatCurrency = new Intl.NumberFormat("id-ID", {
    style: "currency",
    currency: "IDR",
  });

  return formatCurrency.format(amount);
}
