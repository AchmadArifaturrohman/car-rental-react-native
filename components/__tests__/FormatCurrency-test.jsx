import * as React from "react";
import renderer from "react-test-renderer";

import FormatCurrency from "../FormatCurrency";

it(`renders correctly`, () => {
  const tree = renderer.create(<FormatCurrency amount={1000000} />).toJSON();

  expect(tree).toMatchSnapshot();
});
