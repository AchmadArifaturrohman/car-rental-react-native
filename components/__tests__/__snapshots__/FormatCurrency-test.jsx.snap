// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `"RpÂ 1.000.000,00"`;
